src/actions/setup/edit-setup.ts

```diff
@@ -0,0 +1,43 @@
+'use server';
+
+import prisma from '@/lib/prisma';
+import { setupSchema } from '@/schemas/setup-schema';
+import { revalidatePath } from 'next/cache';
+import { cookies } from 'next/headers';
+
+export const editSetup = async ({ id, formData, imageUrl }: { id: string; formData: FormData, imageUrl: string }) => {
+  console.log('Starting editSetup function with data:', { id, formData, imageUrl });
+  const validatedData = setupSchema.parse({
+    title: formData.get('title'),
+    description: formData.get('description'),
+    imageUrl: imageUrl,
```

Comment:
Esto podria simplificarse
```suggestion
    imageUrl,
```